SDL/Segment.cu

```diff
@@ -28,7 +27,8 @@ void SDL::segments::resetMemory(unsigned int nMemoryLocationsx, unsigned int nLo
 void SDL::createSegmentArrayRanges(struct modules& modulesInGPU, struct objectRanges& rangesInGPU, struct miniDoublets& mdsInGPU, uint16_t& nLowerModules, int& nTotalSegments, cudaStream_t stream, const uint16_t& maxPixelSegments)
 {
     /*
-        write code here that will deal with importing module parameters to CPU, and get the relevant occupancies for a given module!*/
+        write code here that will deal with importing module parameters to CPU, and get the relevant occupancies for a given module!
```

Comment:
What is this comment supposed to mean? @YonsiG or @tresreid know maybe?

Reply:
Thanks for the reply, Yanxi!

After investigating a bit, I understood this is a leftover comment for a development that is already done:
https://github.com/SegmentLinking/TrackLooper/blob/master/SDL/Segment.cu#L58-L91
I would remove it.